lib/Transport/NullTransport.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types=1);
+
+namespace Fazland\SkebbyRestClient\Transport;
+
+/**
+ * Null Transport.
+ * Useful for testing purposes.
+ *
+ * @author Massimiliano Arione <garakkio@gmail.com>
+ */
+class NullTransport implements TransportInterface
+{
+    public function executeRequest(string $uri, string $body): string
```

Comment:
Can you add the phpdoc here? 
```php
/**
  * {@inheritdoc}
  */
public function executeRequest(string $uri, string $body): string
```

Reply:
IMHO such phpdoc is useless once you got full type-hinting. Anyway, I understand the request as a matter on consistence with other parts of code (but, again: you should instead remove it everywhere)

Reply:
It is not only for typehintings: if the docs for a method does not need some more explanation from the one defined in the interface we inherit that docs. This is the way we use it 😄